### DISCUSSION TITLE
Repeater header that is populated from a field value

### DISCUSSION THREAD

**Discussion by Zagna** [filamentphp/filament#2429] [Ideas]
With a collapsible repeater that is also orderable, it would be nice to see an indicator of which item you're sorting.
![filament5](https://user-images.githubusercontent.com/14203328/169055037-520b6cb1-82d5-4255-9471-32780e9e0bc6.png)
Instead of that, you could see
![filament4](https://user-images.githubusercontent.com/14203328/169054993-574e940e-1146-4ba5-b04c-3bad9e0a05dc.png)
Something like `->collapsedHeaderField('name')`, where you set the field, the value of which would be used for the title.
I tried tinkering with something... and in order to walk through `childComponents`, the method would need to be after `->schema()`?
And I'm not sure how to properly walk through the components recursively to add some attributes through `extraInputAttributes`.

**Comment by madsem** [comment]
Was just  searching for the same feature, would be awesome to have this.

**Comment by danharrin** [comment]
We have this feature in the latest version

```php
->itemLabel(fn ($state) {
    return $state['name'];
})
```

Make sure the name field is `lazy()` or `reactive()`
